apps/wfo-ui/pages/new-subscription/[processName].tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { useRouter } from 'next/router';
+
+const NewSubscriptionPage = () => {
```

Comment:
I think we are creating a new process here that doesn't necessarily has to be a subscription so the better name would be new-process/[[processName]. Same goes for the other filenames below that contain 'subscription'

Reply:
Renamed to "start-workflow/[workflowName]"